.github/CODEOWNERS

```diff
@@ -400,8 +400,118 @@
+/x-pack/plugins/security_solution/public/common/components/events_viewer @elastic/security-threat-hunting-investigations
+/x-pack/plugins/security_solution/public/common/components/markdown_editor @elastic/security-threat-hunting-investigations
+/x-pack/plugins/security_solution/public/detections/components/alerts_kpis @elastic/security-threat-hunting-investigations
+/x-pack/plugins/security_solution/public/detections/components/alerts_table @elastic/security-threat-hunting-investigations
+/x-pack/plugins/security_solution/public/detections/components/alerts_info @elastic/security-threat-hunting-investigations
+/x-pack/plugins/security_solution/public/resolver @elastic/security-threat-hunting-investigations
+/x-pack/plugins/security_solution/public/timelines @elastic/security-threat-hunting-investigations
+
+/x-pack/plugins/security_solution/server/lib/timeline @elastic/security-threat-hunting-investigations
+
+## Security Solution sub teams - Threat Hunting Explore
+/x-pack/plugins/security_solution/common/search_strategy/security_solution/hosts @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/common/search_strategy/security_solution/matrix_histogram @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/common/search_strategy/security_solution/network @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/common/search_strategy/security_solution/user @elastic/security-threat-hunting-explore
+
+/x-pack/plugins/security_solution/cypress/integration/cases @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/cypress/integration/host_details @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/cypress/integration/hosts @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/cypress/integration/network @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/cypress/integration/overview @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/cypress/integration/pagination @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/cypress/screens/hosts @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/cypress/screens/network @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/cypress/tasks/hosts @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/cypress/tasks/network @elastic/security-threat-hunting-explore
+
+/x-pack/plugins/security_solution/public/common/components/charts @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/public/common/components/header_page @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/public/common/components/header_section @elastic/security-threat-hunting-explore
+/x-pack/plugins/security_solution/public/common/components/hover_actions @elastic/security-threat-hunting-explore
```

Comment:
Only other thing I noticed was this one is the `use_hover_actions` file in this folder actually comes from timelines. If there can be multiple ownership, it would be good to add investigations here as well